src/greeklt/greek_iso_transliteration.py

```diff
@@ -0,0 +1,272 @@
+def greek_iso_transliteration(string: str):
+#   This should all probably be a CSV or something
+    el_low = [
```

Comment:
Instead of lists, it would be prefferable for readabilty reasons to store our letters in dictionaries, eg. {"β": "b"}. And they could all be moved in a seperate file later on, along with other functions' dictionaries.

Reply:
Done! Letters are in a dictionary; as a result I was able to simplify the code a lot. Thanks for the advice! I'd really love to quote some relevant pieces of code but I have no idea how; sorry!